src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -229,24 +247,6 @@ const schema = {
             properties: {
               name: {
                 type: 'string'
-              },
-              address: {
```

Comment:
I wanted to remove the whole `ptsdIncident` and `secondaryPtsdIncident`, but that's out of the scope of this PR, I'm afraid. :disappointed:

Instead, I removed the `address` here because it's not being used (and actually wasn't before, but that's beside the point).